jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -8,6 +8,20 @@ pipeline {
     }
 
     stages {
+        stage('Process manifest') {
+            steps {
+                cleanWs()
+                script {
+                    final Map msg = readJSON text: CI_MESSAGE
+                    sh 'git clone https://github.com/integr8ly/integreatly-operator.git'
```

Comment:
Cloning this repo will be done in it's own stage here https://github.com/integr8ly/ci-cd/pull/414/files#diff-9d245d98e5f80fe8a65190c6e62116d5R18-R23